src/main/java/net/pms/util/OpenSubtitle.java

```diff
@@ -974,53 +882,37 @@ protected static String findImdbIdByFileHash(
 		if (!subtitles.isEmpty()) {
 			result = subtitles.get(0).getIdMovieImdb();
 			if (isNotBlank(result)) {
-				LOGGER.debug(
-					"OpenSubtitles SearchSubtitles returned IMDB ID {} for \"{}\" using file hash \"{}\"",
-					result,
-					resource.getName(),
-					fileHash
-				);
+				LOGGER.debug("OpenSubtitles SearchSubtitles returned IMDB ID {} for \"{}\" using file hash \"{}\"", result,
+					resource.getName(), fileHash);
 				return result;
 			}
 		}
 
-		// Use the less accurate checkMovieHash2 if no subtitles are registered for the file hash
+		// Use the less accurate checkMovieHash2 if no subtitles are registered
+		// for the file hash
 		Map<String, List<CheckMovieHashItem>> results = checkMovieHash2(fileHash);
 		List<CheckMovieHashItem> items = results.isEmpty() ? null : results.get(fileHash);
 		if (items == null || items.isEmpty()) {
-			LOGGER.debug(
-				"OpenSubtitles CheckMovieHash2 returned no IMDB ID for \"{}\" using file hash \"{}\"",
-				resource.getName(),
-				fileHash
-			);
+			LOGGER.debug("OpenSubtitles CheckMovieHash2 returned no IMDB ID for \"{}\" using file hash \"{}\"", resource.getName(),
+				fileHash);
 			return null;
 		}
 		if (prettifier == null) {
-			LOGGER.warn(
-				"OpenSubtitles CheckMovieHash returned {} result{} which can't be verified because the prettifier is null",
-				items.size(),
-				items.size() > 1 ? "s" : ""
-			);
+			LOGGER.warn("OpenSubtitles CheckMovieHash returned {} result{} which can't be verified because the prettifier is null",
+				items.size(), items.size() > 1 ? "s" : "");
```

Comment:
```suggestion
			LOGGER.warn(
				"OpenSubtitles CheckMovieHash returned {} result{} which can't be verified because the prettifier is null",
				items.size(),
				items.size() > 1 ? "s" : ""
			);
```